view/adminhtml/ui_component/algolia_algoliasearch_indexing_queue_listing.xml

```diff
@@ -106,6 +106,17 @@
                 </item>
             </argument>
         </column>
+        <column name="class">
+            <argument name="data" xsi:type="array">
+                <item name="options" xsi:type="object">Algolia\AlgoliaSearch\Model\Source\JobClasses</item>
+                <item name="config" xsi:type="array">
+                    <item name="component" xsi:type="string">Magento_Ui/js/grid/columns/select</item>
```

Comment:
I think adding this class field as a filter in the listing component could be helpful here.

Reply:
Good catch, I forgot it indeed!